ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -82,7 +91,30 @@ public class BottomSheetViewController: UIViewController {
 
         updateResizingHandleViewAccessibility()
     }
+    
+    public override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        if animator != nil {
```

Comment:
not only check this, we need to check if the animation is running

Reply:
will update with animator?.state == .active
earlier I tried animator?.isRunning it was returning false always.

printed the logs in handle pan APIs also, animator?.isRunning is printing false value.